README.md

```diff
@@ -1,254 +1,149 @@
-    <img src="https://github.com/bebasid/bebasid/blob/master/dev/resources/logo-atharva-cloud-whitepng.png?raw=true#gh-dark-mode-only" width="330" target="_blank" alt="Website Provider Cloud Atharva">
-</td><tr>
-  <th>https://www.atharva.co.id/</th>
-<th>https://www.atharva.cloud</th>
-</tr>
-</tbody>
-</table>
-    <i>Penyedia VPS yang berorientasi pada kebebasan dan netral untuk kebutuhannya.</i></br>
-    <m>Dimulai dari <b>Rp. 50.000,00</b></m>.</br>
-    <a href="https://www.atharva.cloud/index.php?rp=/store/cloud-vm-lite">Lihat paket langganan VPS selengkapnya</a>.
-</div>
-<hr>
-<p align="center"><code>KINI Memperkenalkan:</code></br></br>
-    <img src="https://github.com/bebasid/bebasid/blob/master/dev/resources/logo-black.svg#gh-light-mode-only" alt="bebasID White Logo" width="445">
-    <img src="https://github.com/bebasid/bebasid/blob/master/dev/resources/logo.svg#gh-dark-mode-only" alt="bebasID Black Logo" width="445"></br>
-    <i>Membantumu berselancar dengan bebas dan tak ada habisnya!</i>
-</p>
+#### Bahasa / Language
+| <img src="https://em-content.zobj.net/thumbs/120/twitter/351/flag-indonesia_1f1ee-1f1e9.png" width="16"/> **Indonesia** | <img src="https://github.com/twitter/twemoji/blob/master/assets/svg/1f1fa-1f1f8.svg" width="16"/> [English](README.en.md) |
+|-|-|
+
 <p align="center">
-    <a href="#"><img src="https://img.shields.io/static/v1?label=tahun%20diluncurkan:&message=2019&color=yellowgreen&style=plastic%22"></a>
-    <a href="https://github.com/bebasid/bebasid/actions?query=workflow%3AValidate"><img src="https://img.shields.io/github/actions/workflow/status/bebasid/bebasid/validate.yml?branch=master&logo=github&logoColor=fff&label=validasi" alt="Status Hosts"></a>
-    <img src="https://img.shields.io/github/size/bebasid/bebasid/releases/hosts.svg?label=ukuran" alt="Ukuran rilis"></br>
-    <a href="#"><img src="https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Fbebasid%2Fbebasid_fGHyh&count_bg=%234572CD&title_bg=%23555555&icon_color=%23E7E7E7&title=penonton%3A+%28hari%20ini%2Ftotal%29&edge_flat=false"/></a></br>
-    <a href="https://github.com/bebasid/bebasid/blob/master/LICENSE"><img src="https://img.shields.io/badge/lisensi-MIT-green"></a>
+    <img src="dev/resources/logo-black.svg#gh-light-mode-only" alt="BebasID Logo White" width="445">
+    <img src="dev/resources/logo.svg#gh-dark-mode-only" alt="BebasID Logo Black" width="445"><br>
+    <i>Membantumu berselancar dengan bebas dan tak ada habisnya!</i>
 </p>
+
+<a href="LICENSE">
+    <img alt="Static Badge" src="https://img.shields.io/badge/MIT-white?style=for-the-badge&logo=andela&logoColor=white&label=Lisensi&labelColor=222">
+</a>
+
+<img alt="Static Badge" src="https://img.shields.io/badge/2019-white?style=for-the-badge&logo=fireship&logoColor=white&label=Diluncurkan&labelColor=222">
+
 <br>
 
-<p align="center">
-<code>Server Komunitas:</code></br>
```

Comment:
Maybe try adding it back, but putting back on the hidden. on contact. Like click this button to view for community server

Reply:
I dont think a dropdown would look good squished between repo info and donation badges